### DISCUSSION TITLE
Do I need `eslint-plugin-markdown` if I am using `eslint-plugin-mdx`?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [mdx-js/mdx#2275] [Q&A] [answered]
Does `eslint-plugin-mdx` build on top of `eslint-plugin-markdown`?

It seems to me that I don't need both.

**Comment by ChristianMurphy** [comment, marked as answer]
Heya @PaulRBerg! 👋 

`eslint-plugin-mdx` and `eslint-plugin-markdown` are not built on each other and serve different goals.

`eslint-plugin-mdx` checks the JSX and Markdown content of a document directly.
`eslint-plugin-markdown` checks JavaScript inside of code fences, and does not check the surrounding markdown.

Depending on your needs you could use one or both in a project.
